src/bindings/jna/libelektra/src/main/java/org/libelektra/plugin/PropertiesStorage.java

```diff
@@ -1,6 +1,12 @@
 package org.libelektra.plugin;
 
-import java.io.*;
+import static org.libelektra.Elektra.KeyNewArgumentFlags.KEY_END;
+
+import java.io.BufferedInputStream;
```

Comment:
Are they used?

Reply:
What exactly do you mean by they?

Reply:
The import statements (there are also several in the following lines).

Reply:
yes, all stated imports are used.